tests/sles4sap/publiccloud/qesap_terraform.pm

```diff
@@ -51,34 +54,26 @@ sub set_var_output {
 
 =head2 create_ansible_playbook_list
 
-    Detects HANA/HA scenario from openQA variables and creates "ansible: create:" section in config.yaml file.
+    Detects HANA/HA scenario from openQA variables and returns a list of ansible playbooks to include
+    in the "ansible: create:" section of config.yaml file.
 
 =cut
 
-sub create_playbook_section {
+sub create_playbook_section_list {
     # Cluster related setup
     my @playbook_list;
     my @hana_playbook_list;
 
-    # Add registration module as first element- "QESAP_SCC_NO_REGISTER" skips scc registration via ansible
-    unless (get_var("QESAP_SCC_NO_REGISTER")) {
-        my $reg_playbook = "registration.yaml -e reg_code=" . get_required_var("SCC_REGCODE_SLES4SAP") . " -e email_address=''";
-        push(@playbook_list, $reg_playbook);
-    }
-
+    # Add registration module as first element - "QESAP_SCC_NO_REGISTER" skips scc registration via ansible
+    push @playbook_list, 'registration.yaml -e reg_code=' . get_required_var('SCC_REGCODE_SLES4SAP') . " -e email_address=''"
+      unless (get_var('QESAP_SCC_NO_REGISTER'));
 
     # SLES4SAP/HA related playbooks
     if ($ha_enabled) {
-        push(@hana_playbook_list, "pre-cluster.yaml");
-        push(@hana_playbook_list, "sap-hana-preconfigure.yaml -e use_sapconf=" . set_var_output("USE_SAPCONF", "true"));
-        push(@hana_playbook_list, "cluster_sbd_prep.yaml") if (check_var('FENCING_MECHANISM', 'sbd'));
-        push(@hana_playbook_list, "sap-hana-storage.yaml");
-        push(@hana_playbook_list, "sap-hana-download-media.yaml");
-        push(@hana_playbook_list, "sap-hana-install.yaml");
-        push(@hana_playbook_list, "sap-hana-system-replication.yaml");
-        push(@hana_playbook_list, "sap-hana-system-replication-hooks.yaml");
-        push(@hana_playbook_list, "sap-hana-cluster.yaml");
-
+        push @hana_playbook_list, 'pre-cluster.yaml', 'sap-hana-preconfigure.yaml -e use_sapconf=' . set_var_output('USE_SAPCONF', 'true');
```

Comment:
Personally, one playbook inclusion for line is more readable

Reply:
I don't think it's efficient to send so many `push` commands on the same list in succession when fewer commands accomplish the same task.

WDYT about:
```
push @hana_playbook_list,
    'pre-cluster.yaml',
    'sap-hana-preconfigure.yaml -e use_sapconf=' . set_var_output('USE_SAPCONF', 'true');
```
This way preserves the one-playbook-per-line while also not calling `push` on the same list N times consecutively.

Reply:
I agree, and having it in multiple lines makes locating and adding/removing pushes easier